query_test.go

```diff
@@ -668,6 +675,13 @@ SELECT ARRAY_CONCAT_AGG(x) AS array_concat_agg FROM (
 				[]interface{}{nil, int64(1), int64(2), int64(3), int64(4), int64(5), int64(6), int64(7), int64(8), int64(9)},
 			}},
 		},
+		{
+			name:  "array_concat_agg with null in order by",
+			query: `WITH toks AS (SELECT ['1'] AS x, '1' as y UNION ALL SELECT ['2', '3'], null) SELECT ARRAY_CONCAT_AGG(x ORDER BY y) FROM toks`,
+			expectedRows: [][]interface{}{{
+				[]interface{}{"3", "2", "1"},
```

Comment:
I tried this query in the BigQuery Console and got results `2, 3, 1` .
What do you think about this?

Reply:
`2, 3, 1` seems correct. I'll take a look at what's going on.